docs/admin/iceberg-server.md

```diff
@@ -0,0 +1,1351 @@
+  <td>
+    The URI of the Gravitino server address.
+    This is only applicable when `catalog-config-provider` is `dynamic-config-provider`.
+  </td>
+  <td>(none)</td>
+  <td>No</td>
+  <td>`0.7.0-incubating`</td>
+</tr>
+<tr>
+  <td><tt>gravitino.iceberg-rest.gravitino-metalake</tt></td>
+  <td>
+    The metalake name that `dynamic-config-provider` used to request to Gravitino.
+    Only applicable when `catalog-config-provider` is `dynamic-config-provider`.
+  </td>
+  <td>(none)</td>
+  <td>No</td>
+  <td>`0.7.0-incubating`</td>
+</tr>
+<tr>
+  <td><tt>gravitino.iceberg-rest.catalog-cache-eviction-interval-ms</tt></td>
+  <td>Catalog cache eviction interval.</td>
+  <td>`3600000`</td>
+  <td>No</td>
+  <td>`0.7.0-incubating`</td>
+</tr>
+</tbody>
+</table>
+
+```text
+gravitino.iceberg-rest.catalog-cache-eviction-interval-ms = 300000
+gravitino.iceberg-rest.catalog-config-provider = dynamic-config-provider
+gravitino.iceberg-rest.gravitino-uri = http://127.0.0.1:8090
+gravitino.iceberg-rest.gravitino-metalake = test
+```
+
+### Other Apache Iceberg catalog properties
+
+You can add other properties defined in
+[Iceberg catalog properties](https://iceberg.apache.org/docs/1.6.1/configuration/#catalog-properties).
+The `clients` property for example:
+
+<table>
+<thead>
+<tr>
+  <th>Configuration item</th>
+  <th>Description</th>
+  <th>Default value</th>
+  <th>Required</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td><tt>gravitino.iceberg-rest.clients</tt></td>
+  <td>The client pool size of the catalog.</td>
+  <td>`2`</td>
+  <td>No</td>
+</tr>
+</tbody>
+</table>
+
+:::info
+`catalog-impl` has no effect.
+:::
+
+### Event listener
+
+Gravitino generates pre-event and post-event for table operations
+and provide a pluggable event listener to allow you to inject custom logic.
+For more details, please refer to [Event listener configuration](./server-config.md#event-listener-configuration).
+
+### Audit log
+
+Gravitino provides a pluggable audit logging mechanism, please refer to
+[Audit log configuration](./server-config.md#audit-log-configuration).
+
+### Apache Iceberg metrics store configuration
+
+Gravitino provides a pluggable metrics store interface to store and delete Iceberg metrics.
+You can develop a class that implements `org.apache.gravitino.iceberg.service.metrics.IcebergMetricsStore`
+and add the corresponding jar file to the Iceberg REST service CLASSPATH directory.
+
+<table>
+<thead>
+<tr>
+  <th>Configuration item</th>
+  <th>Description</th>
+  <th>Default value</th>
+  <th>Required</th>
+  <th>Since Version</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+  <td><tt>gravitino.iceberg-rest.metricsStore</tt></td>
+  <td>The Iceberg metrics storage class name.</td>
+  <td>(none)</td>
+  <td>No</td>
+  <td>`0.4.0`</td>
+</tr>
+<tr>
+  <td><tt>gravitino.iceberg-rest.metricsStoreRetainDays</tt></td>
+  <td>
+    The days to keep Iceberg metrics in store.
+    The value not greater than 0 means retain forever.
```

Comment:
A value not greater than 0 means retained forever. But this may be better "A negative value means retained forever."